recipes/pymca-zocalo/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pymca-zocalo" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 27100359d8a3e3c6ec8d0bc582116deaccb1ffe467388976284c603b94a04488
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - pymca
+    - python
+    - setuptools
+    - workflows >=1.7
+    - zocalo
+    - xraylib
+
+test:
+  imports:
+    - pymca_zocalo
+
+about:
+  home: "https://github.com/DiamondLightSource/python-zocalo-pymca"
+  license: BSD
+  license_family: BSD
```

Comment:
Please specify which BSD license and use the proper SPDX identifier (e.g., `BSD-3-Clause`).

Reply:
Fixed